app/components/bali/notification/component.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Bali
+  module Notification
+    class Component < ApplicationViewComponent
+      attr_reader :options
+
+      #
+      # Notification Component with different type of notification.
+      #
+      # @param [<Symbol>] type This adds a class for the notification: :success, :danger
+      # @param [<Integer>] miliseconds_to_close How long the notification will be shown.
+      # @param [<Hash>] options This adds a custom attributes to the component.
+
+      def initialize(type: :success, miliseconds_to_close: 3000, fixed: true, **options)
```

Comment:
miliseconds_to_close se me hace muy largo no? mejor usar la misma nomenclatura que en el controlador de JS

```suggestion
      # Notification Component with different type of notification.
      #
      # @param [Symbo] type This adds a class for the notification: :success, :danger
      # @param [Integer] delay How long the notification will be shown.
      # @param [Hash] options This adds a custom attributes to the component.
      def initialize(type: :success, delay: 3000, fixed: true, **options)
```